components/basecomponents/MemoryGame/MemoryGameCore.tsx

```diff
@@ -4,9 +4,9 @@ import { useTranslation } from 'next-i18next';
 import Card from './MemoryGameCard';
 import { TranslationJSON } from 'utils/Phrase';
 import { getCountryVariant } from 'utils/locales';
-import phrases_CZ from 'data/translations/cs/memory-game.json';
-import phrases_PL from 'data/translations/pl/memory-game.json';
-import phrases_SK from 'data/translations/sk/memory-game.json';
+import phrases_CS from './memory-game-cs.json';
+import phrases_PL from './memory-game-pl.json';
+import phrases_SK from './memory-game-sk.json';
 import createTimer from './createTimer';
```

Comment:
přesunuto z data/translations do složky komponenty kůli odlišné struktuře, vlastně to tam asi ani úplně nepatřilo